frontend/views/components/menu/MenuParent.vue

```diff
@@ -24,17 +24,20 @@ export default ({
     }
   },
   methods: {
-    handleTrigger () {
-      this.config.Menu.isActive = true
+    handleToggle (e) {
+      if (this.config.Menu.isActive === e.target.open) return
+      this.config.Menu.isActive = e.target.open
       this.$emit(this.config.Menu.isActive ? 'menu-open' : 'menu-close')
     },
+    handleTrigger () {
+      this.$refs.details.open = true
+    },
     handleSelect (itemId) {
       this.closeMenu()
       this.$emit('select', itemId)
     },
     closeMenu () {
-      this.config.Menu.isActive = false
-      this.$emit('menu-close')
```

Comment:
Maybe `menu-close` custom event here should be kept? (because menu can also be closed via `handleSelect()` above this, not only via `@toggle` event)

Reply:
Marking this as resolved as it looks like it's working fine (see the sample provided, with the toggle event firing every time there's a change, whether user- or programatically-triggeed). If you still think this is an issue, please unresolve.